src/main/java/com/google/cloud/tools/managedcloudsdk/install/Downloader.java

```diff
@@ -87,15 +86,14 @@ public void download() throws IOException, InterruptedException {
           // update progress
           totalBytesRead += bytesRead;
           long bytesSinceLastUpdate = totalBytesRead - lastUpdated;
-          if (totalBytesRead == contentLength || bytesSinceLastUpdate > updateThreshold) {
-            messageListener.message(
-                String.valueOf(totalBytesRead) + "/" + String.valueOf(contentLength));
+          if (totalBytesRead == contentLength || bytesSinceLastUpdate > UPDATE_THRESHOLD) {
+            messageListener.message(".");
```

Comment:
Wouldn't this write a ton of dots if the size is very large?

Reply:
Yeah, I think that's fine, we're just trying to show progress.